src/org/exist/xquery/functions/util/FunctionFunction.java

```diff
@@ -25,6 +25,7 @@
 import org.apache.logging.log4j.Logger;
 import org.exist.dom.QName;
 import org.exist.xquery.*;
+import org.exist.xquery.Module;
```

Comment:
In Java 9 there is a `java.lang.Module`, so we have to explicitly import our one now.